enterprise-suite/console-api/prometheus.yml

```diff
@@ -200,6 +192,10 @@ scrape_configs:
       - source_labels: [pod, es_workload]
         regex: (.+);
         target_label: es_workload
+      - source_labels: [node]
+        target_label: node_name
+      - regex: 'node'
```

Comment:
this needs to be deprecated slowly, so UI can switch to node_name. After UI switches, we can remove it here